microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -9,12 +9,26 @@ ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, edge, ICoreWebView2, ICoreWebView2Controller, Selenium, Microsoft Edge Driver
 ---
-# Automate and test WebView2 with Microsoft Edge Driver
+# Automate and test WebView2 apps with Microsoft Edge Driver
+<!-- Should this article be titled:
+# Automate and test WebView2 with Microsoft Edge Driver, Selenium, and WebDriver
+this filename is webdriver.md, should the title include "WebDriver"?
+-->
 
-Because WebView2 uses the Microsoft Edge \(Chromium\) web platform, WebView2 developers can take advantage of standard web tooling for debugging and automation.  Selenium is one such tool.  It implements the W3C [WebDriver][W3cWebdriver2] API.  You can use Selenium to create automated tests to simulate user interactions.
+<!-- todo: link from some WebDriver articles to here -->
 
-Get started with the following steps.
+This article explains how to automate and test your WebView2 app with Microsoft Edge Driver, by using the Selenium framework for browser test automation.
 
+This article provides instructions for using the Selenium framework and C#, but you can use any library, framework, and programming language that supports WebDriver.  To accomplish the same tasks using a WebDriver testing framework other than Selenium, consult the official documentation for your framework of choice.
+
+To create automated tests to simulate user interactions in your WebView2 app, you can use Microsoft Edge Driver.  The W3C WebDriver protocol allows programs to control the behavior of web browsers.  Microsoft Edge Driver is Microsoft's implementation of the WebDriver protocol, specifically for Microsoft Edge.  Test authors write tests that use WebDriver commands that Microsoft Edge Driver receives.  Microsoft Edge Driver then sends the WebDriver commands to Microsoft Edge.
```

Comment:
```suggestion
To create automated tests that simulate user interactions for your WebView2 app, you can use Microsoft Edge Driver. 
 Microsoft Edge Driver is Microsoft's implementation of the W3C WebDriver protocol.  The W3C WebDriver protocol allows programs to control the behavior of web browsers.  Test authors write tests that use WebDriver commands to tell the browser to perform specific actions.  Microsoft Edge Driver receives these commands and then asks the browser to perform the requested action.  Microsoft Edge Driver supports automating both the Microsoft Edge Browser, and WebView2 apps.
```

Reply:
I thought it was a little jarring to define the W3C WebDriver protocol before mentioning that Microsoft Edge Driver is an implementation of WebDriver, so I switched these sentences around and made a few minor edits here.